### ISSUE TITLE
Enable Authentication for SSO Users

### ISSUE THREAD

**Issue by bxbrenden** [nicohaenggi/SafariBooks-Downloader#78]
## TL;DR
Please consider adding support for SSO users that do not have a password

## Background
The way the downloader works currently relies upon both a username and a password to authenticate. This makes sense, but not all users that have a paid account use a password. In my case, my company has Oreilly linked to their Single-Sign-On server. As soon as I enter my email address, the password input field is hidden, and a button appears that redirects me to my company's SSO server. Once I authenticate, I am returned to Oreilly and can access books.

## Feature Request
It would be great if the `README.md` file explained how to retrieve the Bearer Authorization Token manually from browser developer tools or HTTP responses and if SafariBooks-Downloader allowed you to enter that as a password or token in the command-line arguments. Something along the lines of:
```bash
safaribooks-downloader -b 9781484224427 -u nicohaenggi --token <Bearer_Auth_Token> -o /Users/nicohanggi/Desktop/testbook.epub
```
